build.sbt

```diff
@@ -445,6 +445,14 @@ val mtagsSettings = List(
         .cross(CrossVersion.for3Use2_13)
         .exclude("org.scala-lang", "scala-reflect")
         .exclude("org.scala-lang", "scala-compiler")
+        .exclude(
+          "com.lihaoyi",
+          "geny_2.13"
+        ) // avoid 2.13 and 3 on the classpath since we rely on it directly
+        .exclude(
+          "com.lihaoyi",
+          "sourcecode_2.13"
+        ) // avoid 2.13 and 3 on the classpath since it comes in via pprint
```

Comment:
These seem to causing issues with 1.6.x but not 1.5.x which is interesting. Either way in 1.6.x we get errors about multiple binary version on the classpath since scalameta is bringing in the 2.13 versions whereas we also directly rely on the 3 artifacts in here as well.